src/main/kotlin/edu/agh/susgame/dto/rest/model/Lobby.kt

```diff
@@ -7,10 +7,22 @@ import kotlinx.serialization.Serializable
 data class LobbyId(val value: Int)
 
 @Serializable
-data class Lobby(
+data class LobbyPin(val value: String)
+
+@Serializable
+data class LobbyRow(
+    val id: LobbyId,
+    val name: String,
+    val isPinSetUp: Boolean,
+    val maxNumOfPlayers: Int,
+    val playersWaitingCount: Int,
+)
+
+@Serializable
+data class LobbyDetails(
     val id: LobbyId,
     val name: String,
+    val isPinSetUp: Boolean,
```

Comment:
wsm to tutaj jest bez sensu bo to dostajesz z `get game` a żeby dostawć zwrotkę z get game to musisz podać PIN XD

Reply:
I mean, da się to wywnioskować z kontekstu, ale nie mazwałbym tego "Bez sensu".
To pole kosztuje nas 1 bit danych, a dzięki temu mamy pełnię informacji zwracaną przez endpoint

Reply:
ID to akurat zły przykład bo ono jest często kluczem, a to jest pochodna.

Reply:
Zgadzam się, że tutaj nas to nie boli ale co do zasady nalży sie zastanawiac czy ma to sens i co najmniej być tego świadomym.